### PR TITLE
Added documentation regarding Jumpstart Setup

### PR DESCRIPTION
**Pull request by cjwest**
# READY FOR REVIEW

# Summary
- TL;DR - what's this PR for? 
I added some information I've learned about running `jumpstart-setup.sh`.

# Needed By (Date)
- When does this need to be merged by? 
When more folks start using this script

# Urgency
- How critical is this PR?
This might be useful, but is not critical

# Steps to Test

1. Read documentation
2. Verify it looks right.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules? 
No

# Associated Issues and/or People

- Anyone who should be notified? 
@jbickar 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by jbickar** [comment]
The status says "READY/NOT READY FOR REVIEW". Which is it?

**Comment by cjwest** [comment]
Hi @jbickar,
Thank you for calling my attention to the PR status. It is indeed Ready for Review.

**Review by jbickar — APPROVED** [review]

**Comment by jbickar** [comment]
Thanks @cjwest !
